docker-compose.yml

```diff
@@ -1,6 +1,19 @@
 version: '3'
 
 services:
+    reporter:
+        image: liberoadmin/reviewer-reporter:latest
+    environment:
+        MAILER_AUTH_USER: 
+        MAILER_AUTH_PASS:
+        MAILER_HOST:
+        MAILER_PORT:
+        PGHOST:
+        PGUSER:
+        PGPASSWORD:
+        PGDATABASE:
+        PGPORT:
```

Comment:
ah, https://github.com/elifesciences/elife-xpub-deployment/blob/develop/.env#L7 is wrong (typo). Separate problem

Reply:
Changes made locally pending libero/reviewer-reporter#2